components/Admin/Detail/index.vue

```diff
@@ -0,0 +1,142 @@
+              <jds-icon size="14px" name="pencil" />
+            </div>
+            Ubah Data
+          </div>
+        </template>
+      </BaseButton>
+    </div>
+    <div class="admin-detail__content">
+      <h1 class="admin-detail__content-header">
+        Detail Administrator
+      </h1>
+      <jds-simple-table class="admin-detail__content-table">
+        <thead>
+          <tr>
+            <th>Profil Administrator</th>
+            <th />
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td width="200">
+              <strong>Foto Profil</strong>
+            </td>
+            <td>
+              <img width="90" height="90" class="rounded-full object-cover w-[90px] h-[90px]" :src="item.avatar.path" :alt="item.name">
+            </td>
+          </tr>
+          <tr>
+            <td><strong>Nama</strong></td>
+            <td>{{ item.name || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Email</strong></td>
+            <td>{{ item.email || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Role</strong></td>
+            <td class="capitalize">
+              {{ item.role || '-' }}
+            </td>
+          </tr>
+          <tr>
+            <td><strong>Status</strong></td>
+            <td>{{ item.is_active ? 'Aktif' : 'Non-Aktif' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+
+      <jds-simple-table class="admin-detail__content-table">
+        <thead>
+          <tr>
+            <th>Log User</th>
+            <th />
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td width="200">
+              <strong>Dibuat pada</strong>
+            </td>
+            <td>{{ formattedDate(item.created_at) }}</td>
+          </tr>
+          <tr>
+            <td><strong>Terakhir diupdate</strong></td>
+            <td>{{ formattedDate(item.updated_at) }}</td>
+          </tr>
+          <tr>
+            <td><strong>Login terakhir</strong></td>
+            <td>{{ formattedDate(item.last_login_at) }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+    </div>
+    <AdminEdit :show="showEditAdmin" :item="mItem" @stored="refreshData" @close="showEditAdmin = false" />
+  </div>
+</template>
+
+<script>
+import { formatInTimeZone } from 'date-fns-tz'
+import { id } from 'date-fns/esm/locale'
+
+export default {
+  name: 'ComponentAdminDetail',
+  props: {
+    item: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data () {
+    return {
+      showEditAdmin: false,
+      mItem: {}
+    }
+  },
+  watch: {
+    item: {
+      handler (val) {
+        this.mItem = {
+          ...val
+        }
+      },
+      immediate: true,
+      deep: true
+    }
+  },
+  methods: {
+    refreshData () {
+      this.$emit('updated')
+    },
+    formattedDate (date) {
+      if (date) {
+        return formatInTimeZone(date, 'Asia/Jakarta', 'dd MMMM yyyy HH:mm:ss', { locale: id })
```

Comment:
Just curious, have u tried to format the date using JavaScript without the lib? Or it's too complex?

Reply:
there is a built-in javascript method for this case called `date.toLocaleDateString`

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

but If there is a need for other date-formatting cases `date-fns` will surely help us a lot 👍🏼

Reply:
I haven't try it yet, wdyt is better method to format datetime UTC timezone?

Reply:
I found a way to format it
```
formattedDate (date) {
    const newDate = new Date(date)
    const options = { year: 'numeric', month: 'long', day: 'numeric', hour: '2-digit', minute: '2-digit' }
    const formatDate = newDate.toLocaleDateString('id', options).replace(/\./g, ':')
    return formatDate
}
```

wdyt? should I replace it? @bangunbagustapa

Reply:
Your implementation using JavaScript looks good, you can create an abstraction based on that in case it will be used by another file.

Why I ask this question because I believe we can implement it without using any lib, but if it's too complex or not maintainable, you can consider using the lib.

It depends on the need, but in my personal opinion, if we can handle and create an abstraction or utility function for formatting date without the lib, then we don't have to install the lib. But if there will be a complex date and time format and many components have different needs, you can consider using the lib to create an abstraction.

It's up to you and the team @yoslie 😁 ✌️

Reply:
for now I think we don't need lib and it is the only component that use it, and we can use manual formatter so [here is the changes]([refactor: change date formatter method, remove date-fns](https://github.com/jabardigitalservice/dedi-cms/pull/35/commits/78ec9ca27eed5d95786f6d37c6be126d2bb0811e))